templates/view-request/main.tpl

```diff
@@ -57,35 +57,29 @@
                                         </div>
                                     {/if}
                                     {if $canOauthCreate}
-                                        {if $requestEmailSent}
-                                            <div class="col-md-12 create-button-row {if $currentUser->getCreationMode() !== 1}d-none{/if}" id="createOauth">
+                                        <div class="col-md-12 create-button-row {if $currentUser->getCreationMode() !== 1}d-none{/if}" id="createOauth">
+                                            {if $requestEmailSent}
                                                 <div class="alert alert-warning mb-0">This request has already had an email sent to the requester. Please do a custom close or fall back to manual creation.</div>
-                                            </div>
-                                        {elseif $oauthProblem}
-                                            <div class="col-md-12 create-button-row {if $currentUser->getCreationMode() !== 1}d-none{/if}" id="createOauth">
+                                            {elseif $oauthProblem}
                                                 <div class="alert alert-warning mb-0">There's an issue with your account setup. Please check your OAuth configuration and ensure you've allowed the necessary grants.</div>
-                                            </div>
-                                        {elseif is_array($spoofs) && count($spoofs) > 0}
-                                            <div class="col-md-12 create-button-row {if $currentUser->getCreationMode() !== 1}d-none{/if}" id="createOauth">
+                                            {elseif is_array($spoofs) && count($spoofs) > 0}
                                                 <div class="alert alert-warning mb-0">This request has antispoof hits and cannot be created automatically.</div>
-                                            </div>
-                                        {elseif $requestIsBlacklisted}
-                                            <div class="col-md-12 create-button-row {if $currentUser->getCreationMode() !== 1}d-none{/if}" id="createOauth">
+                                            {elseif $requestIsBlacklisted}
                                                 <div class="alert alert-warning mb-0">This request has title blacklist hits and cannot be created automatically.</div>
-                                            </div>
-                                        {else}
-                                            <div class="col-md-12 create-button-row {if $currentUser->getCreationMode() !== 1}d-none{/if}" id="createOauth">
+                                            {elseif $requestStatus === 'Hospital'}
```

Comment:
Request statuses are defined as constants in `Waca\RequestStatus`. It would be nice if we could use the constant instead of the raw string.

There's basically two ways to do this:
1) Use the constant directly in the template. Since you can't use namespaces in templates (fully-qualified only!), that will be something like `if $requestStatus === Waca\RequestStatus::HOSPITAL`
2) Do the check in the code, and create a new boolean template variable to test in this `{if}`. I'd be tempted to do this next to where `requestIsClosed` is defined in includes/Fragments/RequestData.php, method `setupBasicData`

I have no preference to what you do here - I've used both approaches in different parts of the code.